jboss/container/wildfly/launch/messaging/added/launch/messaging.sh

```diff
@@ -0,0 +1,1053 @@
+    local ctp=5                                  # core thread pool size
+    JBOSS_MESSAGING_ARGS="${JBOSS_MESSAGING_ARGS}
+    -Dactivemq.artemis.client.global.thread.pool.max.size=$mtp
+    -Dactivemq.artemis.client.global.scheduled.thread.pool.core.size=$ctp"
+  fi
+}
+
+# $1 - name - messaging-remote-throughput
+# <!-- ##AMQ_REMOTE_CONNECTOR## -->
+function generate_remote_artemis_remote_connector() {
+    echo "<remote-connector name=\"netty-remote-throughput\" socket-binding=\"${1}\"/>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
+}
+
+# $1 - name - messaging-remote-throughput
+# $2 - server name Optional
+function generate_remote_artemis_remote_connector_cli() {
+    local resource="/subsystem=messaging-activemq"
+    if [ -n "${2}" ]; then
+        resource="${resource}/server=${2}"
+    fi
+    echo "${resource}/remote-connector=netty-remote-throughput:add(socket-binding=\"${1}\")"
+}
+
+# Arguments:
+# $1 - remote context name - default remoteContext
+# $2 - remote host
+# $3 - remote port - 61616
+# <!-- ##AMQ_REMOTE_CONTEXT## -->
+function generate_remote_artemis_naming() {
+    echo "<bindings><external-context name=\"java:global/${1}\" module=\"org.apache.activemq.artemis\" class=\"javax.naming.InitialContext\">
+              <environment>
+                  <property name=\"java.naming.factory.initial\" value=\"org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory\"/>
+                      <property name=\"java.naming.provider.url\" value=\"tcp://${2}:${3}\"/>
+                      <!-- ##AMQ7_CONFIG_PROPERTIES## -->
+              </environment>
+          </external-context>
+          <!-- ##AMQ_LOOKUP_OBJECTS## --></bindings>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
+}
+
+# Arguments:
+# $1 - remote context name - default remoteContext
+# $2 - remote host
+# $3 - remote port - 61616
+function generate_remote_artemis_naming_cli() {
+  local cli_operations
+  IFS= read -rd '' cli_operations << EOF
+
+    if (outcome != success) of /subsystem=naming:read-resource
+      echo You have set MQ_SERVICE_PREFIX_MAPPING environment variable to configure the service name '${service_name}' under '${prefix}' prefix. Fix your configuration to contain naming subsystem for this to happen. >> \${error_file}
+      exit
+    end-if
+
+    if (outcome == success) of /subsystem=naming/binding="java:global/${1}":read-resource
+      echo You have set MQ_SERVICE_PREFIX_MAPPING environment variable to configure the service name '${service_name}' under '${prefix}' prefix. Fix your configuration to not contain a naming binding with name 'java:global/${remote_context_name}' for this to happen. >> \${error_file}
+      exit
+    end-if
+
+    /subsystem=naming/binding="java:global/${1}":add(binding-type=external-context, class=javax.naming.InitialContext, module=org.apache.activemq.artemis, environment={java.naming.provider.url="tcp://${2}:${1}", java.naming.factory.initial=org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory})
```

Comment:
Reviewing I've just found this should be java.naming.provider.url="tcp://${2}:${3}" instead of  java.naming.provider.url="tcp://${2}:${1}"

Reply:
Fixed